### ISSUE TITLE
Warn about targets not matching any OpenAPI operation

### ISSUE THREAD

**Issue by Kallys** [apideck-libraries/portman#391]
Hi!

It would be great if portman can show a warning when a target did not match any OpenAPI operation.
My use case is a big openapi definition split in multiple files, if someone in my team renames an operation id and forget to update portman configuration, associated tests will silently disapears.

Let's say for example someone rename operation `getProducts` to `getAllProducts` in OpenApi but config remains:
```yaml
--
version: 1
tests:
  contentTests:
    - openApiOperationId: getProducts
      responseBodyTests:
        - key: 'products'
          minLength: 1
[...]
```
This content test will silently disapear from generated postman collection (which is, I think, dangerous).

**Comment by nicklloyd** [comment]
👍🏻 I like this idea - we could look at adding a config validation layer that could validate for this, and potentially other config issues.

**Comment by thim81** [comment]
linked to https://github.com/apideck-libraries/portman/issues/632

**Comment by thim81** [comment]
hi @Kallys 
Would this be something you had in mind for the warning?
<img width="968" alt="image" src="https://github.com/user-attachments/assets/6abc04bb-33a7-4b4f-a08e-4871875317f9">

**Comment by Kallys** [comment]
As far as I remember, this sounds great!
I would suggest producing warnings by default to stderr.
And either, providing a verbosity level adjustment option like `--verbose=warn` (by default) and a `--verbose=error` to disable warnings or a file redirection option like `--warn=/dev/null` (to disable warnings) or `--warn=/path/to/warn.log`.
This will help shell scripting for CI jobs.

**Comment by thim81** [comment]
hi @Kallys,

We just released version 1.30.5 of Portman, which includes the warning when a target does not exist in the OpenAPI spec.
Currently it only outputs the [warning using console.log](https://github.com/apideck-libraries/portman/blob/60a76bbfbc3b6b01afcd30f50a071944ec5f7a8d/src/Portman.ts#L690), so I'm not sure what would be the correct way to be used in CI jobs?

If you have suggestions to improvements, let us know in the [discussions](https://github.com/apideck-libraries/portman/discussions) or open a new issue for it.

Closing this item, since a 1st version is released.
